front/src/modules/timesStops/hooks/useTimeStopsColumns.tsx

```diff
@@ -41,25 +41,36 @@ export const useTimeStopsColumns = <T extends TimeStopsRow>(
               ...disabledTextColumn('theoreticalMarginSeconds', t('theoreticalMarginSeconds'), {
                 alignRight: true,
               }),
+              headerClassName: 'padded-header',
               ...fixedWidth(110),
             },
             {
               ...disabledTextColumn('calculatedMargin', t('realMargin'), {
                 alignRight: true,
               }),
-              ...fixedWidth(120),
+              headerClassName: 'padded-header',
+              ...fixedWidth(90),
             },
             {
-              ...disabledTextColumn('diffMargins', t('diffMargins'), { alignRight: true }),
-              ...fixedWidth(120),
+              ...disabledTextColumn('diffMargins', t('diffMargins'), {
+                alignRight: true,
```

Comment:
On the mockup, the margin columns are not aligned on the right but on the left. Same for the arrival column
```suggestion
```

Reply:
I agree, but then on the issue thibaut stated that "All columns but index and CI are right aligned.".

I asked for clarification on this (perhaps he was refering to the headers only? But they also seem to be left aligned on the mockup), but he hasn't had time to answer yet.

Reply:
After discussion with thibaut, durations should be right-aligned and times should be left-aligned for ease of comparison. Text columns will also be left-aligned by default.